hls4ml/converters/onnx_to_hls.py

```diff
@@ -63,6 +63,12 @@ def get_input_shape(graph, node):
     """
     rv = []
     for inp in node.input:
+        # this couple of lines doesn't look very nice but I don't think it would be considered as wrong.
+        # It is necessary for `Resize` node, since RoI input is empty but necessary to specify also scales
+        # array. It might be better handled in QONNX, refers to this issue for more details:
+        # https://github.com/fastmachinelearning/qonnx/issues/150
+        if inp == '':
+            continue
```

Comment:
Wouldn't it be better to put `()` as the shape? Then the inputs aren't shifted.

Reply:
I pushed one commit that addresses this comment, I don't see the code updated from the GH review page but here is the [commit](https://github.com/fastmachinelearning/hls4ml/commit/5757ac62bd54885ac491ce3ba06a86a985dbc1ca)

Reply:
I don't think this is necessary any more.